adam/language_specific/english/english_language_generator.py

```diff
@@ -228,12 +228,24 @@ def _noun_for_object(
                     f"an ontology node currently: {_object}"
                 )
             # TODO: we don't currently translate modifiers of nouns.
-            # Issue: https://github.com/isi-vista/adam/issues/58
 
             # Check if the situation object is the speaker
             if IS_SPEAKER in _object.properties:
                 if syntactic_role_if_known == NOMINAL_SUBJECT:
                     noun_lexicon_entry = I
+                elif any(
```

Comment:
Add a comment here which indicates why we have to special case the subject "I" for the relationship has being the verb in our sentence.

Reply:
Added. Should I make this more concise?

Reply:
@aakum03 I think this is fine.